projects/npm-tools/packages/npm-scripts/package.json

```diff
@@ -44,6 +44,7 @@
 		"liferay-lang-key-dev-loader": "^1.0.3",
 		"liferay-npm-bridge-generator": "2.19.4",
 		"liferay-npm-bundler": "2.19.4",
+		"liferay-r2": "^0.4.4",
```

Comment:
@wincent, do you remember where this currently lives? 😂 

Maybe time to bring this over to `liferay-frontend-projects`?

Reply:
I [commented on this elsewhere](https://github.com/liferay/liferay-frontend-projects/pull/245#discussion_r527554630). It's here: https://github.com/liferay-labs/r2